Lombiq.HelpfulExtensions/Assets/Scripts/target-blank.js

```diff
@@ -9,7 +9,8 @@
                 // In the current script, we are not using them.
                 // eslint-disable-next-line no-script-url
                 !links[i].href.startsWith('javascript:') &&
-                !links[i].href.startsWith('mailto:')) {
+                !links[i].href.startsWith('mailto:') &&
+                links[i].href.length > 0) {
```

Comment:
```suggestion
                links[i].href) {
```
This might be enough.